src/libsyntax/ast.rs

```diff
@@ -220,8 +222,9 @@ impl Path {
     }
 }
 
-/// A segment of a path: an identifier, an optional lifetime, and a set of
-/// types.
+/// A segment of a path: an identifier, an optional lifetime, and a set of types,
+///
+/// e.g. `std`, `String` or `Box<T>`
```

Comment:
Some (most) of the few pre-existing documentations had this pattern, so I tried to stay with it and adapt those that diverted. That being said I'm absolutely open for going with an alternative pattern. I'd just like to have things somewhat uniform.

Thanks for your comments, Pascal. Keep 'em coming! 😃  👍